<documentation title="Function Argument Spacing">
    <standard>
    <![CDATA[
    Function arguments should have one space after a comma.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Single spaces after a comma.">
        <![CDATA[
function foo($bar,<em> </em>$baz)
{
}
        ]]>
        </code>
        <code title="Invalid: No spaces after a comma.">
        <![CDATA[
function foo($bar,<em></em>$baz)
{
}
        ]]>
        </code>
    </code_comparison>
</documentation>
